src/ir/objc.rs

```diff
@@ -96,4 +103,34 @@ impl ObjCInstanceMethod {
     pub fn rust_name(&self) -> &str {
         self.rust_name.as_ref()
     }
+
+    /// Returns the methods signature as FunctionSig
+    pub fn signature(&self) -> &FunctionSig {
+        &self.signature
+    }
+
+    /// Formats the method call
+    pub fn format_method_call(&self, args: &Vec<String>) -> String {
```

Comment:
Also, I'm not super-fond on the name of this method, but I can't suggest anything clearly better so oh well... :)